auth/signup.css

```diff
@@ -0,0 +1,69 @@
+*{
+    margin: 0;
+    padding: 0;
+}
+.nav{
+    display: grid;
+    background-color: #9bb7d4;
+    padding: 0.8rem;
+}
+.name-logo{
+    grid-column: 1/2;
+    grid-row: 1/2;
+    font-size: 25px;
+}
+.nav-ul{
+    grid-column: 2/4;
+    grid-row: 1/2;
+    list-style-type: none;
+    display: flex;
+}
+.nav-a{
+    padding: 2rem;
+    text-decoration: none;
+    color: black;
+    font-size: 20px;
+}
+.nav-side{
+    grid-column: 5/6;
+    grid-row: 1/2;
+    font-size: 20px;
+}
+.nav-join{
+    text-decoration: none; 
+    color: black;
+}
+.mg{
+    margin-left: 2rem;
+}
+.pointer{
+    cursor: pointer;
+}
+.btn{
+     background-color: blue;
+     width: 10%;
+     height: 50px;
+     border-radius: 5px;
+     margin-top: 1rem;
+}
+.signup-bar{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
```

Comment:
These styles can be made common and class name can be used, instead of hard coding everywhere.